GiniVision/Classes/CameraViewController.swift

```diff
@@ -143,7 +143,7 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> ()
         filePickerManager.didPickFile = { [unowned self] document in
             do {
                 try document.validate()
-                self.successBlock?(document)
+                self.successBlock?(document, true)
```

Comment:
Named parameters are so nice for readability (the one thing I really liked about Obj-C). What do you think about using it here? 😍 👉 `self.successBlock(withDocument: document, isImported: true)`

Reply:
Yes, I love them. But unfortunately you cannot use labels on function types 😩